common/resources.yaml

```diff
@@ -18,36 +18,30 @@ groups:
     - null
 
   - data_type_inc: Data
-    name: entities
-    doc: A table for mapping user terms (i.e., keys) to resource entities.
+    name: files
+    doc: A table for storing NWBFile ids used in external resources.
     dtype:
-    - name: keys_idx
-      dtype: uint
-      doc: The index to the key in the 'keys' table.
-    - name: resources_idx
-      dtype: uint
-      doc: The index into the 'resources' table
-    - name: entity_id
+    - name: file_id
       dtype: text
-      doc: The unique identifier entity.
-    - name: entity_uri
-      dtype: text
-      doc: The URI for the entity this reference applies to. This can be an empty string.
+      doc: The file id of the NWBFile that contains the object.
```

Comment:
```suggestion
      doc: The object id of the file that contains the object.
```

Reply:
What do you think about renaming this to file_object_id? In my opinion, file ID is ambiguous. It could also mean the file path or the HDF5 file ID.